AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_ReflectionProbeAdded.py

```diff
@@ -154,44 +202,185 @@ def AtomEditorComponents_ReflectionProbe_AddedToEntity():
+            AtomComponentProperties.reflection_probe('Parallax Correction'), True)
+        Report.result(
+            Tests.parallax_correction,
+            reflection_probe_component.get_component_property_value(
+                AtomComponentProperties.reflection_probe('Parallax Correction')) is True)
+
+        # 9. Toggle Use Baked Cubemap (default True)
+        reflection_probe_component.set_component_property_value(
+            AtomComponentProperties.reflection_probe('Use Baked Cubemap'), False)
+        Report.result(
+            Tests.use_baked_cubemap,
+            reflection_probe_component.get_component_property_value(
+                AtomComponentProperties.reflection_probe('Use Baked Cubemap')) is False)
+
+        reflection_probe_component.set_component_property_value(
+            AtomComponentProperties.reflection_probe('Use Baked Cubemap'), True)
+        Report.result(
+            Tests.use_baked_cubemap,
+            reflection_probe_component.get_component_property_value(
+                AtomComponentProperties.reflection_probe('Use Baked Cubemap')) is True)
+
+        # 10. Toggle Show Visualization (default True)
+        reflection_probe_component.set_component_property_value(
+            AtomComponentProperties.reflection_probe('Show Visualization'), False)
+        Report.result(
+            Tests.show_visualization,
+            reflection_probe_component.get_component_property_value(
+                AtomComponentProperties.reflection_probe('Show Visualization')) is False)
+
+        reflection_probe_component.set_component_property_value(
+            AtomComponentProperties.reflection_probe('Show Visualization'), True)
+        Report.result(
+            Tests.show_visualization,
+            reflection_probe_component.get_component_property_value(
+                AtomComponentProperties.reflection_probe('Show Visualization')) is True)
+
+        # 11. Bake Exposure (float range -20.0 to 20.0)
+        reflection_probe_component.set_component_property_value(
+            AtomComponentProperties.reflection_probe('Bake Exposure'), -20.0)
+        Report.result(
+            Tests.bake_exposure_low,
+            Math_IsClose(
+                reflection_probe_component.get_component_property_value(
+                    AtomComponentProperties.reflection_probe('Bake Exposure')), -20.0))
+
+        reflection_probe_component.set_component_property_value(
+            AtomComponentProperties.reflection_probe('Bake Exposure'), 20.0)
+        Report.result(
+            Tests.bake_exposure_high,
+            Math_IsClose(
+                reflection_probe_component.get_component_property_value(
+                    AtomComponentProperties.reflection_probe('Bake Exposure')), 20.0))
+
+        reflection_probe_component.set_component_property_value(
+            AtomComponentProperties.reflection_probe('Bake Exposure'), 0.0)
+
+        # 12. Inner Extents|Height
```

Comment:
You don't list the components value range on this parameter, but you do have it listed for those above this parameter (Noting default states, or listing the float range on Bake Exposure).

Is there a reason for including the range on some parameters but not all of them?

Reply:
range is constrained by the default size of the added shape which is 8x8x8